src/platform/plugins/shared/cps/kibana.jsonc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "plugin",
+  "id": "@kbn/cps",
+  "owner": "@elastic/kibana-core",
```

Comment:
Should we perhaps share the plugin ownership with @elastic/kibana-esql or the ownership mimicking how Core owns the cloud plugin?